src/track/beatmap.cpp

```diff
@@ -574,6 +574,20 @@ void BeatMap::scale(enum BPMScale scale) {
         // remove every second third and forth beat
         scaleFourth();
         break;
+    case FOURTHIRDS:
+        // introduce two beats into every gap
+        scaleDouble();
```

Comment:
Maybe you can implement `scaleQuadruple()` and use it here instead of two calls to `scaleDouble()`.